core/services.go

```diff
@@ -176,27 +176,28 @@ func (app *Application) endPoll(user *model.User, pollID string) error {
 }
 
 func (app *Application) notifyNotificationsBBForPoll(user *model.User, poll *model.Poll, topic string, operation string, message string) error {
-	recipients, err := app.buildPollNotificationRecipients(user, poll)
-	if err != nil {
-		log.Printf("error while building recipients for notification for poll: %s", err) // dont fail
-		return fmt.Errorf("error while building recipients for notification for poll: %s", err)
-	}
-
-	err = app.notifications.SendNotification(
-		recipients,
-		&topic,
-		"Illinois",
-		message,
-		map[string]string{
-			"type":        "poll",
-			"operation":   operation,
-			"entity_type": "poll",
-			"entity_id":   poll.ID.Hex(),
-			"entity_name": poll.Question,
-		},
-	)
-	if err != nil {
-		log.Printf("error while sending notification for poll end: %s", err) // dont fail
+	if poll.GroupID != nil {
```

Comment:
With this change we are only sending notifications for group polls. Previously we were sending notifications for all polls. 

For non-group polls, we should be sending notifications to everyone in the `ToMembers` list like we were before, and if the `ToMembers` list is empty it looks like we were just sending the notification to everyone subscribed to the `topic`. For example, when the poll ends, everyone on the `topic` was being notified. I am not sure if this is still what we want to do, but I do think we should still be sending some notifications for non-group polls.

Reply:
Hi @shurwit. I did some changes that would address your comment above. Please take a look and let me know if you find anything. Also thanks for the point - it's a nice catch!